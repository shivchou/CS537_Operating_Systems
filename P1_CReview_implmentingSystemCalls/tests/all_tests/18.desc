PART 2: Attempt change to a directory which does not exist, and call getcwd.
