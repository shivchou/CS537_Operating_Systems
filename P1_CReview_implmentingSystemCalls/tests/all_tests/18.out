XV6_TEST_OUTPUT before changing directory.
XV6_TEST_OUTPUT chdir("doesnotexist") failed as expected.
XV6_TEST_OUTPUT getcwd returned / with return value of 0.
